sdk/webpubsub/azure-messaging-webpubsub/src/main/java/com/azure/messaging/webpubsub/implementation/WebPubSubsImpl.java

```diff
@@ -728,272 +884,142 @@ public Response<Boolean> connectionExistsWithResponse(
     }
 
     /**
-     * Close the client connection.
+     * Send content inside request body to the specific connection.
      *
-     * <p><strong>Query Parameters</strong>
+     * <p><strong>Request Body Schema</strong>
      *
-     * <table border="1">
-     *     <caption>Query Parameters</caption>
-     *     <tr><th>Name</th><th>Type</th><th>Required</th><th>Description</th></tr>
-     *     <tr><td>reason</td><td>String</td><td>No</td><td>The reason closing the client connection.</td></tr>
-     *     <tr><td>apiVersion</td><td>String</td><td>Yes</td><td>Api Version</td></tr>
-     * </table>
+     * <pre>{@code
+     * BinaryData
+     * }</pre>
      *
      * @param hub Target hub name, which should start with alphabetic characters and only contain alpha-numeric
      *     characters or underscore.
-     * @param connectionId Target connection Id.
+     * @param connectionId The connection Id.
+     * @param contentType Upload file type. Allowed values: "application/json", "application/octet-stream",
+     *     "text/plain".
+     * @param message The payload body.
      * @param requestOptions The options to configure the HTTP request before HTTP client sends it.
      * @throws HttpResponseException thrown if the request is rejected by server.
-     * @return the completion.
+     * @throws ClientAuthenticationException thrown if the request is rejected by server on status code 401.
+     * @throws ResourceNotFoundException thrown if the request is rejected by server on status code 404.
+     * @throws ResourceModifiedException thrown if the request is rejected by server on status code 409.
+     * @return the {@link Response} on successful completion of {@link Mono}.
      */
     @ServiceMethod(returns = ReturnType.SINGLE)
-    public Mono<Response<Void>> closeConnectionWithResponseAsync(
-            String hub, String connectionId, RequestOptions requestOptions) {
+    public Mono<Response<Void>> sendToConnectionWithResponseAsync(
+            String hub, String connectionId, String contentType, BinaryData message, RequestOptions requestOptions) {
         if (hub == null) {
             return Mono.error(new IllegalArgumentException("Parameter hub is required and cannot be null."));
         }
         if (connectionId == null) {
             return Mono.error(new IllegalArgumentException("Parameter connectionId is required and cannot be null."));
         }
+        if (contentType == null) {
```

Comment:
@srnagar Do we need this kind of validation on client side?

Reply:
`contentType`  is an explicit param in the method, so, checking for `null` is okay for explicit params.

Reply:
ok, it's fine, the reason I ask this is that: I use [empty string](https://github.com/Azure/azure-sdk-for-java/blob/7665d3632adcd9f863f1b59200b12a0582efd111/sdk/webpubsub/azure-messaging-webpubsub/src/main/java/com/azure/messaging/webpubsub/WebPubSubServiceClient.java#L171) `""` as content type to pass this non-null check (I have checked in LIVE test that the content-type passed in through RequestOptions appears in the real HTTP request). So would like to make sure if we really need this validation.